contributors/devel/development.md

```diff
@@ -101,35 +119,56 @@ revising your hypothesis.  The above issues do this transparently, using figures
 and data rather then conjecture. Notice that the problem is analyzed and a
 correct solution is created before a single line of code is reviewed.
 
-## Building Kubernetes with Docker
+## Getting the code
+
+Kubernetes is developed using git for version control.
+
+### GitHub workflow
+
+To check out code to work on, please refer to [this guide](/contributors/guide/github-workflow.md).
+
+For tips, tricks, and common practices used within the Kubernetes project read the [Kubernetes GitHub Cheat Sheet][github-cheat-sheet].
```

Comment:
do you mean the contributor cheat sheet in `contributors/guide/contributor-cheatsheet.md`?

Reply:
We are writing a new [document](https://github.com/kubernetes/community/pull/3213).

Reply:
Are you willing to help me on that part? I want to remove it from the file but I think that it has to go somewhere.

Reply:
See work in https://github.com/kubernetes/community/pull/3436 - that should be great to have in here.